src/osd/OSD.cc

```diff
@@ -2822,6 +2835,10 @@ void OSD::final_init()
                                      " WARNING: Compaction probably slows your requests");
   assert(r == 0);
 
+  r = admin_socket->register_command("get_mapped_pools", "get_mapped_pools",
+                                     asok_hook,
+                                     "dump pools mapped(at least one active PG on) to this OSD.");
```

Comment:
add a space before "(". moreover, could you rephrase this help message? i think

> at least one active PG on

in the parentheses is confusing and hard to read. what is "active" PG in this context? probably we can change it to something like

> dump pools whose PG(s) are mapped to this OSD

?